index.html

```diff
@@ -18,14 +18,132 @@
 
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/main.css">
-
+<!--  <link rel="stylesheet" href="http://use.fontawesome.com/releases/v5.2.0/css/all.css">-->
   <meta name="theme-color" content="#fafafa">
-</head>
+  <script src="https://kit.fontawesome.com/b32a079d69.js" crossorigin="anonymous"></script>
 
+</head>
 <body>
+<!-- Add your site or application content here -->
+
+<main-header></main-header>
+
+<div class="black"></div>
+
+<section class = "welcome">
+  <h1>Welcome</h1>
+  <h2>Car repair services</h2>
+  <p>Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim, quis nostrud nisi ut aliquip ex ea.</p>
+  <button>More info</button>
+  <button>Contact us</button>
+  <img class = "car" src="img/car.png">
+</section>
```

Comment:
HTML w gruncie rzeczy wygląda spoko, ale ja bym dał zdecydowanie więcej div'ów jako kontenerów dla elementów np ta sekcja:

```
<section class="welcome-section">
  <div class="welcome-section__info">
      <h1 class="welcome-section__title">Welcome</h1>
      <h2 class="welcome-section__subtitle">Car repair services</h2>
      <p class="welcome-section__info-paragraph">
         Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, 
         consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim, 
         quis nostrud nisi ut aliquip ex ea.
    </p>
    <div class="welcome-section__button-container">
        <button class="welcome-section__button">More info</button>
        <button class="welcome-section__button">Contact us</button>
    </div>
  </div>
  <div class="welcome-section__image-container">
      <img class = "welcome-section__image" src="img/car.png" alt="car">
  </div>
</section>
```

Taka struktura HTML pozwoliłaby Ci później w CSS na dużo większą kontrolę nad elementami. Przede wszystkim i największy problem jaki sobie stworzyłaś to to, że nie nadawałaś w ogóle klas na elementy. Dosłownie parę na krzyż przez co później w CSS nie mogłaś ostylować konkretnych elementów tylko nadpisywałaś sobie wszystko tagami. Zobacz jak to wygląda wyżej. Każdy element ma klasę więc później w CSS wystarczy tylko się na niego zapiąć i mamy jasny i czysty kod CSS który nam niczego gdzie indziej nie zepsuje.